cra-server/router/routes.js

```diff
@@ -233,9 +235,15 @@ router.get("/open-metadata/admin-services/*", (req, res) => {
     method: "get",
     url: urlRoot + incomingPath,
     httpsAgent: new https.Agent({
+<<<<<<< HEAD
```

Comment:
failed merge here

Reply:
and more later on

Reply:
Yes I havd not pushed up the last changes which addresses these. Sorry for the confusion